Can't find module for Windows 11 Arm JavaScript

Hello everyone!
I use the nestjs library **@nest-modules/mailer^2.0.2** which uses **@css-inline/css-inline 0.14.1**. When building the project I get the following error

```js
node_modules\@css-inline\css-inline\index.js:250
    throw loadError
    ^

Error: Cannot find module '@css-inline/css-inline-win32-arm64-msvc'
```

There is no package for Windows 11 arm? I tried to find and install the dependency manually, but to no avail.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -197,6 +197,9 @@ jobs:
           - host: windows-2022
             build: yarn build
             target: x86_64-pc-windows-msvc
+          - host: windows-2022
+            build: yarn build --target aarch64-pc-windows-msvc
+            target: aarch64-pc-windows-msvc
           - host: ubuntu-22.04
             target: x86_64-unknown-linux-gnu
             docker: ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian
@@ -329,6 +332,11 @@ jobs:
 
       - name: Config yarn (aarch64-linux-android)
         if: matrix.settings.target == 'aarch64-linux-android'
+        run: yarn config set supportedArchitectures.cpu "arm64"
+        working-directory: bindings/javascript
+
+      - name: Config yarn (aarch64-pc-windows-msvc)
+        if: matrix.settings.target == 'aarch64-pc-windows-msvc'
         run: yarn config set supportedArchitectures.cpu "arm64"
         working-directory: bindings/javascript
 

--- a/.github/workflows/javascript-release.yml
+++ b/.github/workflows/javascript-release.yml
@@ -29,6 +29,9 @@ jobs:
           - host: windows-2022
             build: yarn build
             target: x86_64-pc-windows-msvc
+          - host: windows-2022
+            build: yarn build --target aarch64-pc-windows-msvc
+            target: aarch64-pc-windows-msvc
           - host: ubuntu-22.04
             target: x86_64-unknown-linux-gnu
             docker: ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian
@@ -145,6 +148,11 @@ jobs:
 
       - name: Config yarn (aarch64-linux-android)
         if: matrix.settings.target == 'aarch64-linux-android'
+        run: yarn config set supportedArchitectures.cpu "arm64"
+        working-directory: bindings/javascript
+
+      - name: Config yarn (aarch64-pc-windows-msvc)
+        if: matrix.settings.target == 'aarch64-pc-windows-msvc'
         run: yarn config set supportedArchitectures.cpu "arm64"
         working-directory: bindings/javascript
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+### Fixed
+
 - Prioritize `!important` rules when computing element styles. [#398](https://github.com/Stranger6667/css-inline/pull/398)
 
 ## [0.14.2] - 2024-11-11

--- a/bindings/c/CHANGELOG.md
+++ b/bindings/c/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+### Fixed
+
 - Prioritize `!important` rules when computing element styles. [#398](https://github.com/Stranger6667/css-inline/pull/398)
 
 ## [0.14.2] - 2024-11-11

--- a/bindings/javascript/CHANGELOG.md
+++ b/bindings/javascript/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+### Added
+
+- Packages for `win32-arm64-msvc`. [#397](https://github.com/Stranger6667/css-inline/issues/397)
+
+### Fixed
+
 - Prioritize `!important` rules when computing element styles. [#398](https://github.com/Stranger6667/css-inline/pull/398)
 
 ## [0.14.2] - 2024-11-11

--- a/bindings/javascript/npm/win32-arm64-msvc/README.md
+++ b/bindings/javascript/npm/win32-arm64-msvc/README.md
@@ -1,0 +1,3 @@
+# `@css-inline/css-inline-win32-arm64-msvc`
+
+This is the **aarch64-pc-windows-msvc** binary for `@css-inline/css-inline`

--- a/bindings/javascript/npm/win32-arm64-msvc/package.json
+++ b/bindings/javascript/npm/win32-arm64-msvc/package.json
@@ -1,0 +1,31 @@
+{
+  "name": "@css-inline/css-inline-win32-arm64-msvc",
+  "version": "0.14.2",
+  "description": "High-performance library for inlining CSS into HTML 'style' attributes",
+  "keywords": [
+    "css",
+    "html",
+    "email",
+    "stylesheet",
+    "inlining"
+  ],
+  "repository": "https://github.com/Stranger6667/css-inline",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "os": [
+    "win32"
+  ],
+  "cpu": [
+    "arm64"
+  ],
+  "main": "css-inline.win32-arm64-msvc.node",
+  "files": [
+    "css-inline.win32-arm64-msvc.node"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">= 10"
+  }
+}

--- a/bindings/javascript/package.json
+++ b/bindings/javascript/package.json
@@ -23,11 +23,12 @@
     "triples": {
       "additional": [
         "aarch64-apple-darwin",
+        "aarch64-linux-android",
+        "aarch64-pc-windows-msvc",
         "aarch64-unknown-linux-gnu",
         "aarch64-unknown-linux-musl",
         "armv7-unknown-linux-gnueabihf",
         "x86_64-unknown-linux-musl",
-        "aarch64-linux-android",
         "arm-linux-androideabi"
       ]
     }

--- a/bindings/python/CHANGELOG.md
+++ b/bindings/python/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+### Fixed
+
 - Prioritize `!important` rules when computing element styles. [#398](https://github.com/Stranger6667/css-inline/pull/398)
 
 ## [0.14.2] - 2024-11-11

--- a/bindings/ruby/CHANGELOG.md
+++ b/bindings/ruby/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+### Fixed
+
 - Prioritize `!important` rules when computing element styles. [#398](https://github.com/Stranger6667/css-inline/pull/398)
 
 ## [0.14.2] - 2024-11-11

